tests/quantization/test_bitsandbytes.py

```diff
@@ -122,6 +126,30 @@ def test_load_pp_4bit_bnb_model(model_name, description) -> None:
     compare_two_settings(model_name, common_args, pp_args)
 
 
+@pytest.mark.skipif(torch.cuda.device_count() < 2,
+                    reason='Test requires at least 2 GPUs.')
+@pytest.mark.skipif(not is_quant_method_supported("bitsandbytes"),
+                    reason='bitsandbytes is not supported on this GPU type.')
+@pytest.mark.parametrize("model_name, description", models_4bit_to_moe_test)
+@create_new_process_for_each_test()
```

Comment:
```suggestion
@multi_gpu_test(num_gpus=2)
@pytest.mark.skipif(not is_quant_method_supported("bitsandbytes"),
                    reason='bitsandbytes is not supported on this GPU type.')
@pytest.mark.parametrize("model_name, description", models_4bit_to_moe_test)
```
We can use `multi_gpu_test` mark to combine `@create_new_process_for_each_test()` and gpu nums check.

Reply:
I will update this after merging https://github.com/vllm-project/vllm/pull/20692.